worlds/ladx/test/testShop.py

```diff
@@ -0,0 +1,27 @@
+from typing import Optional
+
+from Fill import distribute_planned
+from . import LADXTestBase
+
+
+class PlandoTest(LADXTestBase):
+    options = {
+        "plando_items": [{
+            "items": {
+                "Progressive Sword": 2
+            },
+            "locations": [
+                "Shop 200 Item (Mabe Village)",
+                "Shop 980 Item (Mabe Village)"
+            ]
+        }]
+    }
+        
+    def test_planned(self):
+        """Tests plandoing swords in the shop."""
+        self.multiworld.plando_items[1] = self.options["plando_items"]
+        distribute_planned(self.multiworld)
```

Comment:
overrode the world_setup so that the steps are actually called correctly and added that the locations shouldn't be reachable with starting state. I left the default tests to run because that'll also test the locations are reachable with all_state though I guess I could explicit do that and remove the others.